packages/web/src/pages/app/contributions.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { NextPage } from 'next';
+import { Heading, SimpleGrid, Text, VStack } from '@chakra-ui/react';
+import { AppLayout } from '../../components/Layout';
+import { ContributionsBox } from '../../components/Contributions';
+
+export interface ContributionsList {
+  id: string;
+  nodeID: string;
+  type: string;
+  score: number;
+  contributedAt: Date;
+  description: string | null;
+  url: string;
+}
+
+const Contributions: NextPage = () => {
+  const [Contributions, setContributions] = React.useState<ContributionsList[]>([]);
+
+  React.useEffect(() => {
+    const fetchContributions = async () => {
+      // Get all Contributions
+      const res = await fetch('/api/contributions');
+      const ContributionsList = await res.json();
+
```

Comment:
We need to wrap this in a `try catch` and we also need to make sure that `res.ok` is truthy before continuing. If there's an error, we should display it meaningfully to the user via an [`Alert`](https://chakra-ui.com/docs/feedback/alert)